app/jobs/subscription_placement_job.rb

```diff
@@ -30,11 +29,15 @@ def proxy_orders
   def place_order_for(proxy_order)
     JobLogger.logger.info("Placing Order for Proxy Order #{proxy_order.id}")
     initialise_order(proxy_order)
+    return unless proxy_order.order.present?
+
+    proxy_order.update_column(:placed_at, Time.now)
```

Comment:
We dropped the `zone` bit while moving it. Is https://rails-bestpractices.com/posts/2014/10/22/use-time-zone-now-instead-of-time-now/ not valid anymore?

Reply:
I think `Time.zone.now` is still recommended.

Reply:
Oops yeah that was just an oversight, I just pushed a fix for it.